classes/controllers/grid/users/reviewer/PKPReviewerGridHandler.php

```diff
@@ -1233,6 +1236,33 @@ protected function createMail(Mailable $mailable, string $emailBody, EmailTempla
 
         return false;
     }
+
+    /**
+     * Add the log event
+     *
+     * @param $args array
+     * @param $request PKPRequest
+     *
+     * @return JSONMessage JSON object
+     */
+    public function addLog($args, $request)
+    {
+        $acceptReview = (bool) $request->getUserVar('acceptReview');
+        $decline = !boolval($acceptReview);
+
+        $reviewAssignment = Repo::reviewAssignment()->get($request->getUserVar('reviewAssignmentId'));
```

Comment:
Can we get authorized objects here?
```php
$reviewAssignment = $this->getAuthorizedContextObject(Application::ASSOC_TYPE_REVIEW_ASSIGNMENT);
$submission = $this->getSubmission();
```

Reply:
Thanks @Vitaliy-1 for reminding me. I have removed changes to that file the logic is nolonger used